source/api/fetch.ts

```diff
@@ -50,7 +50,7 @@ export function api(url: string | any, init: any): Promise<any> {
     // Handle failing errors
     if (!response.ok) {
       process.exitCode = 1
-      console.error(`Request failed: ${response}`)
+      console.error(`Request failed: ${JSON.stringify(response, null, 2)}`)
       const msg = response.status === 0 ? "Network Error" : response.statusText
```

Comment:
Would we want to log the JSON response here - `response.json()`? Just a suggestion, could go either way.